JapaneseFilter/sections/specific.txt

```diff
@@ -8598,6 +8598,7 @@ itest.5ch.net##.ranking > div[style^="height: 320px;"]
 itest.5ch.net,itest.bbspink.com##.res_ad
 itest.5ch.net,itest.bbspink.com##div[class^="sproutad_"]
 itest.5ch.net,itest.bbspink.com##[class*="js-ads-content-top_"]
+city.nagaoka.niigata.jp##body > main > div.koukokuWrap
```

Comment:
Looks like this is fine. Please move it to the beginning of the file.
```adblock
city.nagaoka.niigata.jp##.koukokuWrap
```

Reply:
ok